### PR TITLE
EZ-95 Default styling of the components

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

File: modules/ezcontent_paragraphs/modules/ezcontent_card/css/card.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* .paragraph--type--card .field--name-field-title {
+    color:black
+} */
```
Comment (vaibhavjain-in) [inline modules/ezcontent_paragraphs/modules/ezcontent_card/css/card.css]: Remove commented / unused code.

File: modules/ezcontent_paragraphs/modules/ezcontent_referenced_card/ezcontent_referenced_card.module [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * Implements hook_theme().
+ */
+function ezcontent_referenced_card_theme($existing, $type, $theme, $path) {
+  return [
+    'node_article_card' => [
+      'template' => 'node--article--card',
+      'base hook' => 'node'
+    ]
+  ];
+}
```
Comment (vaibhavjain-in) [inline modules/ezcontent_paragraphs/modules/ezcontent_referenced_card/ezcontent_referenced_card.module]: Please help me understand the requirement of this code.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@surbhi90 Can you please fix
1. formatting.
2. remove commented code.

File: modules/ezcontent_paragraphs/modules/ezcontent_card/css/card.css (hunk 1 above)
Comment (surbhi90) [reply modules/ezcontent_paragraphs/modules/ezcontent_card/css/card.css]: @vaibhavjain-in I have made some changes in css in hero media so once these changes will push i have to check whether this change is required or not and after push if this change is not required then will remove it in next PR

File: modules/ezcontent_paragraphs/modules/ezcontent_referenced_card/ezcontent_referenced_card.module (hunk 2 above)
Comment (surbhi90) [reply modules/ezcontent_paragraphs/modules/ezcontent_referenced_card/ezcontent_referenced_card.module]: @vaibhavjain-in I have added this code as we need to overwrite the twig for reference card for bartik theme as we are getting the title twice but right now this code is not working so will either fix this code or will remove this code in next PR

File: modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (vaibhavjain-in) [inline modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module:3]: Please add a new line before this line.

File: modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+ * Implements hook_preprocess_HOOK().
+ */
+function ezcontent_card_preprocess_paragraph__card(&$variables) {
+  $variables['#attached']['library'][] = 'ezcontent_card/ezcontent_card_css';
+}
```
Comment (vaibhavjain-in) [inline modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module]: Please add a line at EOF

File: modules/ezcontent_paragraphs/modules/ezcontent_embed/ezcontent_embed.module [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+function ezcontent_embed_preprocess_paragraph__embed(&$variables) {
+  $variables['#attached']['library'][] = 'ezcontent_embed/ezcontent_embed_css';
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ezcontent_embed_preprocess_block(&$vars) {
+  $bundle = $vars['elements']['content']['#block_content']->bundle();
+  if($bundle === 'embed_block') {
```
Comment (vaibhavjain-in) [inline modules/ezcontent_paragraphs/modules/ezcontent_embed/ezcontent_embed.module]: Please add a space after if statement.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@surbhi90 please cater to minor changes.

File: modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module (hunk 3 above)
Comment (surbhi90) [reply modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module:3]: done

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

File: modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module (hunk 4 above)
Comment (surbhi90) [reply modules/ezcontent_paragraphs/modules/ezcontent_card/ezcontent_card.module]: done

**Pull request merged**